src/content/en/2021/accessibility.md

```diff
@@ -463,66 +461,66 @@ It goes on to say:
 
 ### Requiring information
 
-When web developers gather input from their end users, they need a clear way to indicate what information is optional (the default), and what information is required to proceed. For example, a shipping address is optional if the end user is buying something online that they can download. However, the method of payment is most likely required in order to complete the sale.
+When web developers gather input from their end users, they need a clear way to indicate what information is optional, and what information is required to proceed. For example, a shipping address is optional if the end user is buying something online that they can download. However, the method of payment is most likely required in order to complete the sale.
```

Comment:
Removed "(the default)" as felt that could be confusing for forms that have `required` on all their inputs so the "default" for that fault is required. I do agree that technically it's the default in HMTL but don't think that really matters here and, as I say, thought it could be confusing, so remove. Don't think it changes the sentence much.